lib/commands/connect/export.js

```diff
@@ -13,26 +14,18 @@ module.exports = {
     ],
     needsApp: true,
     needsAuth: true,
-    run: cli.command(function (context, heroku) {
-      api.withUserConnections(context.auth.password, context.app, context.flags, false, heroku).then(function(connections){
-          if (connections.length > 1) {
-            console.log("Multiple connections found. Please use '-r' to specify a single connection by resource name.");
-            api.connection_info(connections);
-          } else {
-            connections.forEach(function(connection) {
-              api.request(context.auth.password, 'GET', '/api/v3/connections/' + connection.id + '/actions/export')
-                .then(function(response) {
-                  let fName = connection.app_name + "-" + (connection.resource_name || '') + ".json";
-                  fs.writeFile(fName, JSON.stringify(response.json, null, 4), function(err) {
-                    if(err) {
-                        return console.log(err);
-                    } else {
-                      console.log("Saved config file: ", fName);
-                    }
-                  });
-                });
-            });
-          }
+    run: cli.command(co.wrap(function* (context, heroku) {
+      let connection = yield api.withConnection(context, heroku);
+      let url = '/api/v3/connections/' + connection.id + '/actions/export';
+      let response = yield api.request(context.auth.password, 'GET', url)
+      let fName = connection.app_name + "-" + (connection.resource_name || '') + ".json";
```

Comment:
That's not a bad idea, but yeah, I'd consider that a new feature for a separate PR.